incubator/brotli/src/main/java/org/glassfish/jersey/message/BrotliEncoder.java

```diff
@@ -0,0 +1,37 @@
+package org.glassfish.jersey.message;
```

Comment:
I do not believe it is a good idea to have the same package name in two modules, and `org.glassfish.jersey.message` is in core-common already.